utils/emoji.go

```diff
@@ -58,3 +61,11 @@ func CreateTestPng(t *testing.T, width int, height int) []byte {
 
 	return buffer.Bytes()
 }
+
+func ReadTestWebp(t *testing.T) []byte {
```

Comment:
There's no need to create a util function yet. We can write this in the caller function.

Reply:
Done.